olm/README.md

```diff
@@ -13,29 +13,26 @@ WARNING: Please make sure to use the precise `v1.7.1` version of the `operator-s
 
 ## Eclipse Che OLM bundles
 
-OLM (operator lifecycle manager) provides ways of installing operators. One of the convenient way how to achieve it is by using OLM bundles. See more about the format: https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md. There two "next" platform-specific OLM bundles for Сhe operator:
+OLM (operator lifecycle manager) provides ways of installing operators. One of the convenient way how to achieve it is by using OLM bundles. See more about the format: https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md. There two OLM bundles:
```

Comment:
I think we also can metion about next-all-namespace bundle.

Reply:
tech-preview channel will removed soon